pandas/core/groupby/ops.py

```diff
@@ -828,11 +832,11 @@ def result_index(self):
         return self.binlabels
 
     @property
-    def levels(self):
+    def levels(self) -> List:
```

Comment:
I think List[int] here as well

Reply:
Could this perhaps be List[Index] since this is [self.binlabels]? It looks like at line 733 we call ensure_index(binlabels) and below here we call binlabels.name. The docstring is weird though ("binlabels : the label list")